DOCSP-35885: Update Flutter Read page with getBacklinks() method (#3172)
mongodb/docs-realm: source/sdk/flutter/crud/read.txt

.. literalinclude:: /examples/generated/flutter/read_write_data_test.snippet.query-all-objects.dart
   :language: dart

> .. _flutter-get-backlinks::

Author: ```suggestion
.. _flutter-get-backlinks:
```

.. literalinclude:: /examples/generated/flutter/read_write_data_test.snippet.query-all-objects.dart
   :language: dart

.. _flutter-get-backlinks:

> Query Linked Objects

Reviewer: I wonder if this section should be "Query an Inverse Relationship" since that's how we refer to it in the data modeling section?

~~~~~~~~~~~~~~~~~~~~

.. versionadded:: 1.9.0

> You can find all objects that link to a given object by calling the

Reviewer: Maybe refer to inverse relationship here?
```suggestion
You can find all objects with an inverse relationship to a given object by calling the
```

.. literalinclude:: /examples/generated/flutter/read_write_data_test.snippet.filter-iterable.dart
   :language: dart

> You can also filter by linking objects using the

Reviewer: In our model data section, we don't refer to linking objects - we refer to inverse relationships or backlinks. We should probably be consistent here with the way we refer to this type of object link in this SDK's docs.

```suggestion
You can also filter by inverse relationships using the
```
